src/components/issues/Card.tsx

```diff
@@ -107,6 +72,34 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
                     Convert to task
                 </button>
             </div>
+            <p className={styles.card__issue_created__by}>
+                Opened on {created} by {issue.user.login}
+            </p>
+            <div className="card__body">
+                <ReactMarkdown className={styles.card__body__description}>
+                    {issue.body ?? 'No description provided'}
+                </ReactMarkdown>
+            </div>
+            {issue.assignee ? 'Assigned to ' + issue.assignee.login : ''}
```

Comment:
I think we can use && here instead of ternary, as we only have to display if there's any assignee otherwise nothing.


I also see that previously, ?: was used, but it makes sense to use && here.